IBAnimatable/MaskType.swift

```diff
@@ -46,23 +44,23 @@ public extension MaskType {
       return
     }
     
-    let nameAndParames = MaskType.extractNameAndParams(from: string)
+    let nameAndParames = MaskType.extractNameAndParams(from: string.lowercased())
```

Comment:
May do `.lowercased()` inside `extractNameAndParams` then all caller don't need to call it?